templates/components/form/fields_macros.html.twig

```diff
@@ -345,6 +346,11 @@
    {{ _self.field(name, field, label, options) }}
 {% endmacro %}
 
+{% macro emailField(name, value, label = '', options = {}) %}
+   {% set options = {'type': 'email'}|merge(options) %}
```

Comment:
```suggestion
   {% set options = {'type': 'email'}|merge(options) %}
```
should not multiple be handled here only aswell?

Reply:
I don't get what you mean, do you have a code example ?